test/index.test.js

```diff
@@ -84,20 +84,61 @@ describe('Wigzo', function() {
       });
 
       it('should call index', function() {
-        analytics.page();
-        analytics.called(window.wigzo.index);
-      });
-
-      it('should pass page name and default properties via page', function() {
-        analytics.page('Name');
-        analytics.called(window.wigzo.index, {
-          title: document.title,
-          url: window.location.href,
-          name: 'Name',
-          path: window.location.pathname,
-          referrer: document.referrer,
-          search: window.location.search
-        });
+        var productData = {
+          product_id: '40',
+          canonicalUrl : 'https://snoopy.wigzopush.com/index.php?route=product/product&amp;product_id=40',
+          imageUrl : 'https://snoopy.wigzopush.com/image/cache/catalog/demo/iphone_1-228x228.jpg',
+          category: 'Mobile Phones',
+          description: 'iPhone is a revolutionary new mobile phone that allows you',
+          language: 'en',
+          name: 'iPhone',
+          brand: 'Apply',
+          price: 18.99,
+          currency: 'usd'
+        };
+
+        var wigzoProduct = {
+          productId: productData.product_id,
+          title: productData.name,
+          price : productData.currency + ' ' + productData.price,
+          category: productData.category,
+          image : productData.imageUrl,
+          canonicalUrl : productData.canonicalUrl,
+          description: productData.description,
+          language: productData.language
+        };
+
+        analytics.track('Product Clicked', productData);
```

Comment:
you want to pass in the options as the 3rd argument.

track takes the event name, properties, and context/options.

so split your non spec'd properties into like a `options` variable where that is: 

```
var options = {
   Wigzo: { // make sure this is capitalized
       image: ....
    }
};

analytics.track('Product Clicked', productData, options);